apps/react/runner.js

```diff
@@ -1,17 +1,25 @@
 const process = require('process');
 const createTestCafe = require('testcafe');
+const path = require('path');
 
 let testCafe;
+
 createTestCafe('127.0.0.1')
     .then(tsc => {
         testCafe = tsc;
-
         const runner = tsc.createRunner();
+
         return runner
-            .src('test.js')
-            .browsers('chrome:headless --disable-gpu --window-size=1200,800')
+            .src(path.join(__dirname, './__test__/*.ts'))
```

Comment:
Do we need these changes to the runner since we are not adding new tests in this PR?

Reply:
reverted